docs/plugin-proposal-pipeline-operator.md

```diff
@@ -18,19 +18,19 @@ Its value is `"hack"` by default.
 
 | Value | Proposal | Version added |
 | ----- | -------- | ------------- |
-| ~~`"minimal"`~~ | [Minimal F#-style pipes](https://github.com/tc39/proposal-pipeline-operator/) | `v7.0.0`
+| `"hack"` | [Hack-style pipes](https://github.com/tc39/proposal-pipeline-operator) | `v7.15.0`
```

Comment:
The hack pipes proposal has been moved to the `proposal-pipeline-operator`, so I changed the `minimal` proposal link to be a snapshot of the tc39 proposal repo.